src/java/org/apache/cassandra/cql3/statements/SelectStatement.java

```diff
@@ -1605,7 +1612,13 @@ private String asCQL(QueryOptions options, ClientState state)
         ColumnFilter columnFilter = selection.newSelectors(options).getColumnFilter();
         StringBuilder sb = new StringBuilder();
 
-        sb.append("SELECT ").append(queriedColumns().toCQLString());
+        String columns = selection.getColumnMapping()
```

Comment:
I was looking for a raw query to log but I was not able to get it. This works only partially. If the query is `select count(*) from ks.tb`, then this will log it as `select count from ks.tb` which missing the asterisk.